### DISCUSSION TITLE
GitHub's .md renderer doesn't see <nobr> tag at all.

### DISCUSSION THREAD

**Discussion by Cnfnbcn** [community/community#56124] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![screenshot-25-22-05-23](https://github.com/community/community/assets/78432987/bacbfa6d-7df2-4dd7-955a-a3ceb59504df)
On the screenshot, in the raw markdown, I have `/[unity-project-root]/Packages/` surrounded with nobr tag. But GitHub doesn't care at all and splits it up still.
![screenshot-26-22-05-23](https://github.com/community/community/assets/78432987/545a216a-4cc8-4756-87f3-e5f86768cc98)

Problem: what I want to achieve is to have a code block (surrounded with `) to be unbreakable - i.e. if there's not enough space on current line, it will go to next line **as a whole**. I tried inserting the unbreakable hyphen, but it doesn't work inside code blocks.
I tried nobr tag, and **it works** in JB Rider as I expect it to. But GitHub doesn't respect it.

**Comment by Remi-Coulom** [comment]
I also would love to find a way to prevent github from breaking lines in the middle of words such as "C++". Is there any workaround?
